src/profile/components/saveIndicator/SaveIndicator.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import commonFormStyles from '../../../common/cssHelpers/form.module.css';
+import { Action } from '../../hooks/useProfileDataEditor';
+
+type Props = {
+  action: Action | undefined;
+  testId: string;
+};
+
+const SaveIndicator = (props: Props): React.ReactElement | null => {
+  const { action, testId } = props;
+  const { t } = useTranslation();
+  if (!action) {
+    return null;
+  }
+  const translationKey =
+    action === 'remove' ? 'notification.removing' : 'notification.saving';
+  return (
+    <div
+      role="alert"
+      className={commonFormStyles.visuallyHidden}
```

Comment:
Is the SaveIndicator hidden from screen?

Reply:
When form is submitted, buttons are disabled while saving. SaveIndicator is just for screen readers to indicate that submit happened and what kind of action (save/remove) is being done.